sklearn/utils/_mocking.py

```diff
@@ -135,3 +135,27 @@ def score(self, X=None, Y=None):
 
     def _more_tags(self):
         return {'_skip_test': True, 'X_types': ['1dlabel']}
+
+
+class _NoSampleWeightWrapper(BaseEstimator):
```

Comment:
Now that the files start with `_`, we don't need the class name to start with and underscore.